uk_bin_collection/uk_bin_collection/councils/BexleyCouncil.py

```
@@ -22,106 +16,69 @@ class CouncilClass(AbstractGetBinDataClass):
-            if driver:
-                driver.quit()
+        user_uprn = kwargs.get("uprn")
+
+        page = f"https://waste.bexley.gov.uk/waste/{user_uprn}"
+
+        headers = {
+            "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36"
+        }
+
+        # First request may trigger async page generation; retry if content not ready
+        for attempt in range(3):
```

Ruff (0.15.9)

[warning] Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)